docs/source/release_notes/2020.2.rst

```diff
@@ -26,11 +28,25 @@ solved. As in the stationary case, the implementation allows for several
 customizations regarding the network architecture and training parameters.
 
 
+Linear PDE-constrained parameter optimization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In `[#1110] <https://github.com/pymor/pymor/pull/1110>`_, some tools for
+PDE-constrained parameter optimization were added. These include partial derivatives
+of the solutions and the output with respect to the parameters.
+
+
+Tutorial: Model order reduction for PDE-constrained optimization problems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The new features that were added in `[#1110] <https://github.com/pymor/pymor/pull/1110>`_
+are gathered in a new tutorial `[#1205] <https://github.com/pymor/pymor/pull/1205>`_ which
+also focuses on general questions and insights to MOR for
+linear PDE-constrained optimization method. 
```

Comment:
This is maybe missing an article? "for a linear PDE..."

Reply:
thx